lib/config/presets/internal/monorepo.ts

```diff
@@ -129,7 +129,10 @@ const repoGroups = {
   'material-components-web':
     'https://github.com/material-components/material-components-web',
   mdx: 'https://github.com/mdx-js/mdx',
-  'material-ui': 'https://github.com/mui-org/material-ui',
+  'material-ui': [
+    'https://github.com/mui-org/material-ui', // Previous organization name (see: https://github.com/mui/material-ui/pull/30944)
```

Comment:
@viceice It sounds like it would be good to add a `replacement` preset for the rename as well?

https://docs.renovatebot.com/presets-replacements/

Reply:
MUI has a [blog post that explains the rename](https://mui.com/blog/material-ui-is-now-mui/).

Reply:
@HonkingGoose let's do it in separate PR, please open a new feature request for it.

Reply:
See issue #14254. 😉

Reply:
Thx @HonkingGoose